src/Microsoft.ML.Core/Data/ISchemaBindableMapper.cs

```diff
@@ -58,12 +58,17 @@ internal interface ISchemaBoundMapper
     /// This interface combines <see cref="ISchemaBoundMapper"/> with <see cref="IRowToRowMapper"/>.
     /// </summary>
     [BestFriend]
-    internal interface ISchemaBoundRowMapper : ISchemaBoundMapper, IRowToRowMapper
+    internal interface ISchemaBoundRowMapper : ISchemaBoundMapper, IRowToRowMapperBase
```

Comment:
>IRowToRowMapperBase [](start = 67, length = 19)

This is what I feel needs to stop here. We have this descent here, but the output schema meaning is rather different.